### ISSUE TITLE
Add symbology to QGIS Resource Sharing plugin

### ISSUE THREAD

**Issue by AuSpotter** [afrigeri/geologic-symbols-qgis#15]
**Is your feature request related to a problem? Please describe.**
Having a variety of QGIS symbology sources makes it difficult for users to see what all is available - they need to search the web, github, etc

**Describe the solution you'd like**
Adding the symbology (github repository) to the Resource Sharing plugin would make this great geological symbols source more readily available, in a central location within QGIS. Very helpfully, other geological groups have added symbology there. https://plugins.qgis.org/plugins/qgis_resource_sharing/

**Comment by gasato** [comment]
I think it is a good idea to add this project to a common community catalog using the Resource Sharing plugin. It will give more visibility to this project.

**Comment by afrigeri** [comment]
Agree, the structure and files for the resource sharing plugin are included (not working yet).  To keep symbols ordered within the same git repo, we need to create a structure of collections corresponding to separate auth/geologic survey/societies.

**Comment by afrigeri** [comment]
from #b439aff92dd17626e62ee032e60c49a5011dcc55 we have now an experimental support for the QGIS Resource Manager Plugin.  

1. install the __resource sharing__  plug-in from the core-plugin list of QGIS
2. add https://github.com/afrigeri/geologic-symbols-qgis.git to the repository list in the resource sharing __Settings__ tab
 ![image](https://user-images.githubusercontent.com/158104/145658382-72aacf55-1d06-4b1d-91ea-b00b16cedcfb.png)
3. go in 'All collections' and search the __gsymblib FGDC__ library, click on the install button
![image](https://user-images.githubusercontent.com/158104/145658445-16e54b67-6298-41a6-845a-df09814fac39.png)

4. wait a bit, and you should have received a message about 400+ SVGs and 1 XML symbols file being loaded
<img src="https://user-images.githubusercontent.com/158104/145658463-01a46a21-afe2-43fb-8ef9-4c6211208795.png" width=50% height=50%>
<img src="https://user-images.githubusercontent.com/158104/145658496-36717676-8181-45d8-b511-ec71c8578cba.png" width=70% height=70%>

The symbols are all ok, __BUT__ all the symbols requiring a pattern are broken, as I did not find a way to generalize the [relative] path from the symbols to the SVG.    

It's an improvement anyway! :)

**Comment by mbodnar-GS** [comment]
Amazing work, if the reference SVG's are embedded using the Style Manager, they should transfer ok, no?

**Comment by afrigeri** [comment]
@mbodnar-GS good point Matthew!  we got to try this.   Originally (manual install) the styles catch the SVG by a relative path, with the resource sharing I did not find a solution yet to go relative path from symbols to SVGs.
